css/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20px;
```

Comment:
```suggestion
  font-size: 1.25rem;
```
Please consider using fluid measurements such as rem. They are much better for responsive design because they are relative measurements. 
px, pixels are absolute measurements. rem is the font size of the root element default size of 16px. When you resize the screen or viewport, the rem adjusts.